### DISCUSSION TITLE
Set ds18b20 as room temperature sensor

### DISCUSSION THREAD

**Discussion by akula3** [rvdbreemen/OTGW-firmware#158] [Q&A] [answered]
Ist that possible instead of outside temperature?
I want to use the otgw in stand-alone mode as room unit

**Comment by rvdbreemen** [comment]
@akula3 you can have multiple additional sensors, to measure temperature. What they mean is up to you, just go here: https://github.com/rvdbreemen/OTGW-firmware/wiki/Using-DS18B20-sensor-with-ESP-firmware

This is the option where you use the ESP to actually have multiple sensors. It's not done by the OpenTherm Gateway PIC, that's another way of doing it.

**Reply by akula3** [reply]
@rvdbreemen , thank you for the feedback!
So, means that, that the temperature value from DS18B20 (connected directly to the OTGW from Nodoshop) does not impact any functions/automations in OTGW for control the boiler/thermostat?

**Comment by rvdbreemen** [comment, marked as answer]
This option to add a DS18B20 to your setup directly to the ESP does not influence other functions. It just gives you the option to add temperature sensors, multiple in fact as they are single wire. On the latest version of hardware you have additional pads on your OTGW PCB to simply wire your sensors.

Make sure, this is NOT the PIC feature, but it's in my firmware. It also just added the temperature as an MQTT message.  So some setup required, but it seems to do for people that have used it. I have had one user connecting 9 different sensors.

**Reply by akula3** [reply]
OK, undersood. Thank you!
